pkgs/development/python-modules/nagiosplugin/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, twine
+, numpy
+, pytest
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "nagiosplugin";
+  version = "1.3.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1vr3zy0zfvbrqc4nf81zxv4gs2q82sv5sjamdm4573ld529mk2nv";
+  };
+
+  buildInputs = [ twine ];
```

Comment:
How is twine used? We basically never add Python modules as `buildInputs`.

Reply:
it is a build time dependency only: https://github.com/mpounsett/nagiosplugin/blob/master/requirements_build.txt